main-project/src/boards/dto/update-board.dto.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  IsBoolean,
+  IsDate,
+  IsNotEmpty,
+  IsNumber,
+  IsString,
+} from 'class-validator';
+
+export class UpdateBoardDto {
+  @IsNotEmpty()
+  @IsString()
+  title: string;
+
+  @IsNotEmpty()
+  @IsString()
+  description: string;
+
+  @IsBoolean()
+  @IsNotEmpty()
+  done: boolean;
```

Comment:
```suggestion
  @IsBoolean()
  @IsNotEmpty()
  isDone: boolean;
```
변수명 앞에 is를 붙이면 Boolean을 의미합니다~ 네이밍 컨벤션 잘 맞춰주는 게 좋습니다!

Reply:
creatdto에서는 수정했었는데 여기선 깜빡했네요;; 수정하겠슴미다 :)